turbonfs/CMakeLists.txt

```diff
@@ -167,10 +177,10 @@ endif()
 find_program(MESON_EXECUTABLE meson QUIET)
 if(NOT MESON_FOUND)
```

Comment:
Meson and ninja not required in case of dynamic linking we can use libfuse.so from installed package for building. We need to check which version it install by default and whether its compatible or not. If not then we can use static linking of libfuse